publish/check-for-published-rpm.sh

```diff
@@ -48,9 +48,16 @@ EOF
 
 # Try to install the new RPMs (don't actually install, only check for errors in
 # case of an install), to see whether they're actually valid.
-if ! xargs -rtd '\n' -a new-rpms.txt yum install -q --assumeno; then
-  echo 'FAILED: could not install new RPMs. See above for errors from yum.'
-  exit 1
+xargs -rtd '\n' -a new-rpms.txt yum install --assumeyes --setopt tsflags=test
+yumerr=$?
+
+if [ $yumerr -gt 0 ]; then
+  echo 'FAILED: could not install new RPMs. See above for errors from yum.' >&2
+else
+  echo 'SUCCESS: no errors reported when installing new RPMs (dry-run only)' >&2
 fi
 
-echo 'SUCCESS: no errors reported when installing new RPMs (dry-run only)' >&2
+# We don't need the downloaded RPMs any more, so clear the cache to avoid
+# accumulating crud.
+yum clean packages || true
```

Comment:
This will still keep them installed, though, no?

Reply:
They are never actually installed in the first place -- `yum install --setopt tsflags=test` makes sure they *could* be installed, downloads the packages, but doesn't actually install them.